bundles/org.openhab.binding.opensprinkler/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -180,15 +181,48 @@
 		<label>Next Duration</label>
 		<description>The duration the station will be opened the next time it is switched on.</description>
 		<category>Time</category>
-		<state readOnly="false" pattern="%.0f min"/>
+		<state readOnly="false" pattern="%.0f min">
+			<options>
+				<option value="15s">15 Seconds</option>
+				<option value="1min">1 Minute</option>
+				<option value="5min">5 Minutes</option>
```

Comment:
Do your options cover all possible values or only few of them ?
Is the pattern still necessary ? For other values ?
I have a doubt about the values in your options with different units. What unit is used when building the channel state in the binding.
Is it something you tested ?
Same comments for the other channel (rainDelay).

Reply:
1. Covers values that most users should be happy with. Metadata can be added to change these default values if they do not suit and I have tested doing this. The issue IMHO is when no defaults are provided the control/widgets default to ANALIZE and showing a graph instead of offering a list of suitable commands to send. For a new openHAB user these changes are far better and also saves time for experienced users if the defaults are suitable.
2. RE pattern: I have removed it for the `nextDuration` as the pattern was not necessary. For the rainDelay channel the API uses hours so I would prefer this gets displayed by default so if people see the TRACE logs they are less likely to get confused. Users can over ride the pattern with metadata. Originally when the binding was merged the ANALIZE feature did not take UOM into account when stored in persistence and this worked around that, not sure if this is fixed now in newer openHAB cores as I have seen it mentioned a few times.
3. The rainDelay has a state that the API will count down in hours. The code does handle all the different units and this is something I have tested for 10+ months now as I have been using Metadata to add the options in different units.

Reply:
I have never played with Number:Time but if you set 24 hours as value, your final value as a string will be 1d and not 24h ? 

And for the other channel, are you sure you will get 1min rather than 60s for example?

Reply:
All I know is that I have tested it, and it works as I don't know how the UOM works in depth internally. When the command is sent, the device is queried and the response back from the device updates the channels state in HOURS as per the pattern. The channel that has had the pattern removed can at times display the result in seconds if for example a time of 4 days is selected it can change to be shown in seconds. Most of the time it displays the selected label as the nextDuration, as the device does not have a state for this. It is only a virtual control used when you turn a watering zone on.

Reply:
I am not clear with that. Can another maintainer please comment & review? @fwolter @cweitkamp @wborn @kaikreuzer

Reply:
I've never tested if a StringType sent to a Channel, defined in state options and the Channel linked to an UoM Item converts correctly when the string is suffixed with the unit. There's a QuantityType constructor that accepts strings in the format defined in above state options. I think it can work and I trust in @Skinah if he says he tested it, that it works.

Reply:
In this case, please merge this PR @fwolter